pkg/controller/controller.go

```diff
@@ -133,6 +133,7 @@ func (c *controller) run(workers int, stopCh <-chan struct{}) {
 	}
 
 	<-stopCh
+	c.started = false
```

Comment:
You need to acuire the startLock before writing this.

Reply:
@JacieChao It seems like it should be:

```
c.startLock.Lock()
defer c.startLock.Unlock()
c.started = false
```

Reply:
fixed